bin_sort/bin_sort.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
```

Comment:
exported type Bucket should have comment or be unexported

*Origin: GoLintBear, Section: `golint`.*